### DISCUSSION TITLE
Access to pictures on external drive

### DISCUSSION THREAD

**Discussion by KilliAisaka** [deckerst/aves#947] [Q&A] [answered]
Hey Everyone,

is it possible to access pictures that are on a via USB-C connected SSD?
i am looking for an option to access the pictures on my Samsung T7 Touch without having to go through the File Manager

**Comment by deckerst** [comment, marked as answer]
Not at the moment. Related issue:
- #366

**Reply by KilliAisaka** [reply]
thank you
